### PR TITLE
Part 1: Task Management application

### PR DESCRIPTION
**Pull request by thiennh-1324**
### Evidence 

[Screencast from 07-23-2024 05:07:35 PM.webm](https://github.com/user-attachments/assets/0e8ea561-2ea0-4cf4-9c41-a5c7bf171136)

image
image
image

### CODE REVIEW

**Comment by thiennh-1324** [comment]
ready

File: src/tasks/tasks.service.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@nestjs/common';
+import { CreateTaskDto } from './dto/create-task.dto';
+import { UpdateTaskDto } from './dto/update-task.dto';
+
+@Injectable()
+export class TasksService {
+  create(createTaskDto: CreateTaskDto) {
+    console.log(createTaskDto);
```
Comment (binhpt-1239) [inline src/tasks/tasks.service.ts:8]: Bỏ console.log này k push lên nhé

File: src/app.service.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class AppService {
+  getHello(): string {
+    return 'Hello World!';
```
Comment (binhpt-1239) [inline src/app.service.ts:6]: Thay bằng i18n test thử luôn Thiện.
Quay anh cái video demo evidence nào

**Comment by thiennh-1324** [comment]
ready

**Review by binhpt-1239 — APPROVED** [review]

**Pull request merged**